### PR TITLE
Chart taglib

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

File: modules/apps/foundation/chart/chart-taglib/src/main/java/com/liferay/chart/taglib/servlet/taglib/soy/base/BaseChartTag.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+	private void _outputStylesheetLink() {
+		OutputData outputData = _getOutputData();
+
+		StringBundler sb = new StringBundler();
+
+		sb.append("<link rel=\"stylesheet\" href=\"");
+		sb.append(PortalUtil.getPathModule());
+		sb.append("/chart-taglib");
+		sb.append("/node_modules/billboard.js@1.1.0/dist/billboard.css\">");
```
Comment (jbalsas) [inline modules/apps/foundation/chart/chart-taglib/src/main/java/com/liferay/chart/taglib/servlet/taglib/soy/base/BaseChartTag.java]: Could we get this path using the `npmResolver`?

File: modules/apps/foundation/chart/chart-taglib/src/main/java/com/liferay/chart/taglib/internal/NPMResolverProvider.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		return _npmResolverProvider._npmResolver;
+	}
+
+	public NPMResolverProvider() {
+		_npmResolverProvider = this;
+	}
+
+	@Reference(unbind = "-")
+	protected void setNPMResolver(NPMResolver npmResolver) {
+		System.err.println("========================> setNPMResolver: "+npmResolver);
```
Comment (jbalsas) [inline modules/apps/foundation/chart/chart-taglib/src/main/java/com/liferay/chart/taglib/internal/NPMResolverProvider.java]: I guess this is not really needed, right? :)

Comment (izaera) [reply modules/apps/foundation/chart/chart-taglib/src/main/java/com/liferay/chart/taglib/internal/NPMResolverProvider.java]: 😬

File: modules/apps/foundation/chart/chart-taglib/src/main/java/com/liferay/chart/taglib/servlet/taglib/soy/base/BaseChartTag.java (hunk 1 above)
Comment (izaera) [reply modules/apps/foundation/chart/chart-taglib/src/main/java/com/liferay/chart/taglib/servlet/taglib/soy/base/BaseChartTag.java]: Yes 👍

**Comment by jbalsas** [comment]
I'll apply changes and send to Brian! 👍

**Pull request closed without merging**